c2cciutils/checks.py

```diff
@@ -126,11 +149,16 @@ def editorconfig(
     args: Namespace,
 ) -> bool:
     """
-    Check the right editorconfig configuration
+    Check the right editorconfig configuration.
```

Comment:
```suggestion
    Check the editorconfig configuration.
```
For sure we check it's right